force-app/main/default/classes/VisualforcePageExerciseController.cls

```diff
@@ -0,0 +1,25 @@
+public class VisualforcePageExerciseController {
+	
+    public List<Account> accounts {get;set;}
+    
+	public VisualforcePageExerciseController() {
+        accounts = [
+            SELECT Id, Name, Description, MessageBody__c
+            FROM Account
+            LIMIT 200
+        ];
+        
+	}
+    public void saveMessage(){
+        List<Message__c> messages = new List<Message__c>();
```

Comment:
esta funcion deberia insertar un mensaje y luego insertar en el for por las cuentas seleccionadas un mensaje en cuenta para hacer una relacion muchos a muchos